modules/utility/init.zsh

```diff
@@ -30,8 +30,15 @@ else
   # colors for ls and completion
   export LSCOLORS='exfxcxdxbxGxDxabagacad'
   export LS_COLORS='di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'
-
-  alias ls='ls -G'
+ 
+  # stock OpenBSD ls does not support colors at all, but colorls does. 
+  if [[ $OSTYPE == openbsd* ]]; then
+    if (( ${+commands[colorls]} )); then
+      alias ls='colorls -G'
+    fi 
+  else  
```

Comment:
Oh, I was looking at the commit and misaligned the termination of the `if`s. I didn't realize that the whole thing was in a conditional. This nesting wasn't an issue before due to the previous structure, so I was just looking at that tidbit. You are correct, the logic is fine. Derp. :)